Open Judge System/Workers/OJS.Workers.ExecutionStrategies/RubyExecutionStrategy.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace OJS.Workers.ExecutionStrategies
+{
+    using System.IO;
+
+    using OJS.Common.Extensions;
+    using OJS.Workers.Checkers;
+    using OJS.Workers.Common;
+    using OJS.Workers.Executors;
+
+    public class RubyExecutionStrategy : ExecutionStrategy
+    {
+        protected const string SubmissionFileName = "_$Submission.rb";
```

Comment:
You don't need a named submission file, since it's better to use File.Helpers to get a temp file anyway.